### ISSUE TITLE
Webhook Model missing the version field

### ISSUE THREAD

**Issue by awharkrider** [zendesk/sunshine-conversations-java#10]
The webhook model appears to be missing the expected version field.

Current webhook for smooch-java 4.0 https://github.com/smooch/smooch-java/blob/master/docs/Webhook.md

Name | Type | Description 
-- | -- | --
id | String | The webhook ID, generated automatically. |  
target | String | URL to be called when the webhook is triggered. |  
triggers | List<String> | An array of triggers you wish to have the webhook listen to. If unspecified the default trigger is message. |  
secret | String | Secret which will be transmitted with each webhook invocation and can be used to verify the authenticity of the caller.

Expected as indicated on https://docs.smooch.io/rest/#manage-webhooks: 
```
{
    "webhook": {
        "_id": "55c8d9758590aa1900b9b9f6",
        "triggers": ["message:appUser"],
        "secret": "8564b3e6a8b20a4bdb68b05d9ea97aace9bc5936",
        "target": "http://example.com/callback",
        "version": "v1.1"
    },
}
```

The version field could be an essential field for migrating/upgrading to v1.1.

**Comment by mspensieri** [comment]
Good catch! We'll make sure to add that field

**Comment by mspensieri** [comment]
This is now fixed in v4.1.0

**Comment by awharkrider** [comment]
Thanks! Appreciate the quick response.
